src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -26,6 +26,20 @@ public boolean connect(String database, String user, String password) {
         }
     }
 
+    /**
+     * Method for inserting data into a table.
```

Comment:
It's not correct javadoc. You should desribe parameters and return value using the block tags @param and @return

Reply:
I wrote this not as a javadoc, but as a description of the method.
From the point of view of javadoc, this entry is useless

Reply:
Well, javadoc is exactly the thing which describes a method behavior, besides you used javadoc formatting, that is a reason why I pay your attention on that... In case of using simple comments I wouldn't say anything. IMHO